bridges/HetznerBridge.php

```diff
@@ -0,0 +1,33 @@
+<?php
+class HetznerBridge extends BridgeAbstract {
+
+	const MAINTAINER = 'jlelse';
+	const NAME = 'Hetzner News';
+	const URI = 'https://www.hetzner.com/news/';
+	const DESCRIPTION = 'Get news from Hetzner';
+	const CACHE_TIMEOUT = 21600; // 6h
+
+	const PARAMETERS = array();
+
+	public function collectData(){
+		$html = getSimpleHTMLDOM(self::URI) or returnServerError('Could not request Hetzner News: ' . self::URI);
```

Comment:
Move `or returnServerError` to a new line to improve readability.
```suggestion
		$html = getSimpleHTMLDOM(self::URI) 
			or returnServerError('Could not request Hetzner News: ' . self::URI);
```